docs/CUSTOMIZING.rst

```diff
@@ -65,42 +148,102 @@ Similar to configuration for disabling redaction, if you wish to hide certain me
 
 If these metadata items contain unexpected values (e.g., text where a number was expected), they will be visible and available for redaction.
 
+.. code-block:: python
+
+  [wsi_deid]
+  ...
+  hide_metadata_keys = {
+      "^internal;openslide;openslide\.level\[": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
+  hide_metadata_keys_format_aperio = {
+      "^internal;openslide;(openslide\.comment|tiff\.ImageDescription)$": "",
+      "^internal;openslide;aperio\.(Original(Height|Width)|Left|Top|Right|Bottom|LineArea(X|Y)Offset|LineCameraSkew|Focus Offset|StripeWidth|DisplayColor)": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
+  hide_metadata_keys_format_hamamatsu = {
+      "^internal;openslide;hamamatsu\.((AHEX|MHLN|YRNP|zCoarse|zFine)\[|(X|Y)OffsetFromSlideCentre|ccd.(width|height)|(focalplane|slant)\.(left|right)(top|bottom)|stage.center)": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
+  hide_metadata_keys_format_philips = {}
```

Comment:
Let's add the isyntax block.

Also, can we add something like:

--

To show ALL possible metadata, set each of these values to ``{}``:

--

and then a code block illustrating that.

Reply:
be52e04